### DISCUSSION TITLE
failed to install SO 2.4 RC2

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#10982] [2.4] [answered]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/29997801/770ebae4-7222-4710-9f03-adb19cf67769)

**Comment by dougburks** [comment, marked as answer]
Can you provide more information?

Looking at your screenshot, it appears that your installation succeeded and services are starting. Please see https://docs.securityonion.net/en/2.4/console.html for more information about the console messages.
